lib/rouge/lexers/qsim.rb

```diff
@@ -22,8 +22,8 @@ def any(expressions)
           /#{expressions.join('|')}/
         end
 
-        rule any(operators), Keyword
-        rule any(registers), Name::Builtin::Pseudo
+        rule any(operators), Operator
```

Comment:
Hmm, I understand that there is some ambiguity here - are mnemonics operators or keywords?. 

I don't have a definitive semantic answers here, but from a pragmatic point of view, I'd consider then as Keywords, since ACE editor does the same, and we need to maximize consistency among them. 

So, please, keep mnemonics as Keywords. Treating registers as variables is fine